examples/OTEL-logging/otel-logging.mdx

```diff
@@ -0,0 +1,147 @@
+
+```
+OTEL_EXPORTER_OTLP_ENDPOINT=https://api.braintrust.dev/otel
+OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer <Your API Key>, x-bt-parent=project_name:<Your Project Name>"
+```
+
+Replace `<Your API Key>` with your Braintrust API key, and `<Your Project Name>` with the name of the project in Braintrust where you’d like to store your logs. 
+
+<Callout type="info">
+The `x-bt-parent` header sets the trace's parent project or experiment. You can use a prefix like `project_id:`, `project_name:`, or `experiment_id:` here, or pass in a [**span slug**](/docs/guides/tracing#distributed-tracing) (`span.export()`) to nest the trace under a span within the parent object.
+</Callout>
+
+You’ll also need to set up an [**OpenTelemetry Protocol Exporter**](https://opentelemetry.io/docs/languages/js/exporters/) (OTLP) to send traces to Braintrust. Create a new file called `instrumentation.ts` to register OTel:
+
+```tsx
+import { registerOTel } from "@vercel/otel";
+
+export function register() {
+  registerOTel({
+    serviceName: "multi-step-tool-calls-demo",
+  });
+}
+```
+
+Then, configure Next.js telemetry by adding the following to your `next.config.mjs` file:
+```
+nextConfig.experimental = {
+  instrumentationHook: true,
+};
+```
+
+You can then use the `experimental_telemetry` option to enable telemetry on supported AI SDK function calls. In your `route.ts` file, add this import statement:
+
+```tsx
+import { getOrders, getTrackingInformation } from "@/components/data";
+```
+
+and add the `experimental_telemetry` parameter to your LLM call: 
+
+```tsx
+export async function POST(request: Request) {
+  const { messages } = await request.json();
+
+  const stream = await streamText({
+    model: openai("gpt-4o"),
+    system: `\
+      - you are a friendly package tracking assistant
+      - your responses are concise
+      - you do not ever use lists, tables, or bullet points; instead, you provide a single response
+    `,
+    messages: convertToCoreMessages(messages),
+    maxSteps: 5,
+    tools: {
+      listOrders: {
+        description: "list all orders",
+        parameters: z.object({}),
+        execute: async function ({}) {
+          const orders = getOrders();
+          return orders;
+        },
+      },
+      viewTrackingInformation: {
+        description: "view tracking information for a specific order",
+        parameters: z.object({
+          orderId: z.string(),
+        }),
+        execute: async function ({ orderId }) {
+          const trackingInformation = getTrackingInformation({ orderId });
+          await new Promise((resolve) => setTimeout(resolve, 500));
+          return trackingInformation;
+        },
+      },
+    },
+    experimental_telemetry: {
+      isEnabled: true,
+      functionId: "multi-step-tool-calls-demo",
+      metadata: { foo: "bar" },
```

Comment:
up to you, but maybe you call out that you can add metadata here to make your LLM calls easier to track (and they'll end up as metadata in the Braintrust span)